c7n/resources/kms.py

```diff
@@ -47,6 +54,8 @@ def augment(self, resources):
                     self.log.warning(
                         "Access denied getting tags for key:%s",
                         key_id)
+                else:
+                    raise
```

Comment:
@jhnlsn - per a conversation this morning with @kapilt, I added in the logic to re-raise exceptions when the tags fail also.